src/WebJobs.Extensions.DocumentDB/Config/DocumentDBConfiguration.cs

```diff
@@ -56,6 +55,11 @@ internal static string Resolve(string key)
 
                 if (string.IsNullOrEmpty(value))
                 {
+                    value = ConfigurationManager.AppSettings[key];
+                }
+
+                if (string.IsNullOrEmpty(value))
+                {
                     value = Environment.GetEnvironmentVariable(key);
                 }
             }
```

Comment:
It returns null. This isn't really a 'Resolve' method (which is why I renamed it). This is treating the string strictly as a connectionstring/appsetting/environmentvar. If it's not there, it returns null.